productivity-suite/app/legacy-app/index.html

```diff
@@ -159,20 +159,32 @@
         border-radius: 1px;
       }
 
+      piercing-fragment-host {
+        max-width: 80vw;
```

Comment:
thanks to this we don't get the fragment outlines overlap the legacy app ones on smaller screens

Reply:
and in any case avoid effects like this:
![Screenshot 2022-11-04 at 17 42 26](https://user-images.githubusercontent.com/61631103/200041260-5d386406-3490-4b6f-86df-e8a579ee34e1.png)